Viper/Viper/CountryList/Router/CountryListRouterProtocol.swift

```diff
@@ -0,0 +1,17 @@
+// CountryListRouterProtocol.swift
+// Copyright © SolovevAA. All rights reserved.
+
+import UIKit
+
+/// Протокол роутера экрана со списком стран
+protocol CountryListRouterProtocol {
+    // MARK: - Public Properties
```

Comment:
```suggestion
    // MARK: - Public Properties
```
мне в протоколах в прошлый раз стирали все марки